pkgs/development/python-modules/pyopencl/default.nix

```diff
@@ -60,32 +64,29 @@ buildPythonPackage rec {
     pybind11
   ] ++ os-specific-buildInputs;
 
-  propagatedBuildInputs = [
-    appdirs
-    cffi
-    decorator
-    mako
+  dependencies = [
     numpy
```

Comment:
Speaking of `numpy`, could it be this package supports both numpy versions? Perhaps this shouldn't be propagated.

Reply:
a topic for another PR